src/test/regress/upgrade/custom_citus.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
```

Comment:
we should probably not have these scripts live under `upgrade/`

Reply:
hmm one option could be to change the name `upgrade` as well

Reply:
can't/shouldn't we have a new folder `regress/arbitrary_config` and files under there?